src/Illuminate/Database/Grammar.php

```diff
@@ -208,14 +208,19 @@ public function isExpression($value)
     }
 
     /**
-     * Get the value of a raw expression.
+     * Transforms expressions to their scalar types.
      *
-     * @param  \Illuminate\Database\Query\Expression  $expression
-     * @return mixed
+     * @param  scalar|\Illuminate\Contracts\Database\Query\Expression  $expression
+     * @return scalar
      */
     public function getValue($expression)
     {
-        return $expression->getValue();
+        $value = $expression;
```

Comment:
Is this just to rename the variable?

If so, isn't it easier to rename the method's parameter?

Reply:
The parameter should be an expression as defined by the variable name. But some core function of Laravel ignores that and calls it with a value. So it exists to make it more clear.